tests/testthat/_snaps/simulate.md

```diff
@@ -0,0 +1,39 @@
+# simulate errors if distribution-free variables are not fixed
+
+    Code
+      sims <- simulate(m)
+    Error <simpleError>
+      the target <greta_array>s are related to variables that do not have
+      distributions so cannot be sampled
+
+# simulate errors if a distribution cannot be sampled from
+
+    Code
+      sims <- simulate(m)
+    Error <simpleError>
+      sampling is not yet implemented for "hypergeometric" distributions
+
+# simulate errors nicely if nsim is invalid
+
+    Code
+      simulate(m, nsim = 0)
+    Error <simpleError>
+      value must be a positive integer
```

Comment:
should say `nsim`?

Reply:
This is that issue with the argument `nsim` not being passed down to the fundamental checking function. Not sure how to address this.